### PR TITLE
Update pom.xml JaCoCo version to 0.8.8

### PR DESCRIPTION
**Pull request by v-niranjan**
Updating JaCoCo version to support code build on [Automate Java container deployments with Azure Pipelines](https://docs.microsoft.com/en-us/learn/modules/deploy-java-containers/4-deploy-java-container )

### CODE REVIEW

**Comment by v-niranjan** [comment]
Hi @juliakm , 

PR is ready for Technical review.

File: pom.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
       <plugin>
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
-        <version>0.7.9</version>
+        <version>0.8.2</version>
```
Comment (juliakm) [inline pom.xml]: Is there a reason why this version is better than some of the newer ones? https://mvnrepository.com/artifact/org.jacoco/jacoco-maven-plugin

**Review by juliakm — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (v-niranjan) [reply pom.xml]: Yes, v0.8.8 works fine with https://docs.microsoft.com/en-us/learn/modules/deploy-java-containers/
Tested on https://github.com/v-niranjan/mslearn-java-containers/pull/6

**Review by juliakm — APPROVED** [review]
Thank you! This will help with future-proofing!

**Comment by v-niranjan** [comment]
Hi @juliakm , 

I don't have access to merge this branch. Can you please merge it to the repo and as discussed kindly rename the default branch from master-->main 

Regards, 
@v-niranjan

**Pull request merged**